Chat/LoginWindow.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<Window x:Class="Chat.LoginWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Chat"
+        mc:Ignorable="d"
+        Title="LoginWindow" Height="200" Width="200
+        ">
+    <Grid Background="Orange"> 
+        <TextBlock x:Name="LoginTB" Background="LightGray" Margin="55,30,55,0" TextWrapping="Wrap" VerticalAlignment="Top" Height="18"/>
+        <TextBlock x:Name="PasswordTB" Background="LightGray" Margin="55,60,55,0" TextWrapping="Wrap" VerticalAlignment="Top" Height="18"/>
```

Comment:
Change Name on "Password_TB". Don't use TextBlock in "Passwords",  try use PasswordBox, and add property PasswordChar="*".